src/portal/DefaultPortal.sol

```diff
@@ -60,13 +60,24 @@ contract DefaultPortal is Initializable, AbstractPortal, IERC165Upgradeable {
       block.timestamp,
       attestationPayload.expirationDate,
       false,
+      0,
+      0,
       1,
       attestationPayload.attestationData
     );
 
     attestationRegistry.attest(attestation);
   }
 
+  /**
+   * @notice Revokes an attestation for given identifier and can replace it by an other one
+   * @param attestationId the attestation ID to revoke
+   * @param replacedBy the replacing attestation ID
```

Comment:
Can we add a code comment to say that the zero address can be used to indicate that the attestation is being revoked without being replaced